extras/gke-autopilot/hpa/frontend.yaml

```
@@ -0,0 +1,46 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_boa_kubernetes_manifests_frontend_horizontal_pod_autoscaler_frontend]
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: frontend
+spec:
+  behavior:
+    scaleUp:
+      stabilizationWindowSeconds: 0
+      policies:
+        - type: Percent
+          value: 100
+          periodSeconds: 5
+      selectPolicy: Max
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: frontend
+  minReplicas: 1
+  maxReplicas: 25
+  metrics:
+    - type: External
+      external:
+        metric:
+          name: loadbalancing.googleapis.com|https|request_count
+          selector:
+            matchLabels:
+              resource.labels.forwarding_rule_name: <forwarding_rule_name>
```

```suggestion
              resource.labels.forwarding_rule_name: FORWARDING_RULE_NAME
```

Using current variable convention. Also makes sed stuff much less of a headache :)

---

Good point.